pom.xml

```diff
@@ -670,17 +669,10 @@
                 <groupId>io.swagger.core.v3</groupId>
                 <artifactId>swagger-jaxrs2</artifactId>
                 <version>${swagger.version}</version>
-            </dependency>
-
-            <!-- ref: https://github.com/swagger-akka-http/swagger-scala-module -->
-            <dependency>
-                <groupId>com.github.swagger-akka-http</groupId>
-                <artifactId>swagger-scala-module_${scala.binary.version}</artifactId>
-                <version>${swagger.scala.module.version}</version>
```

Comment:
Is it we do not use `Option` type as the value of annotation, so we can remove this dependency safely ?

Reply:
not quite sure